appliances/VRouter/DHCP4v2/main.rb

```diff
@@ -121,6 +121,7 @@ def install(initdir: '/etc/init.d')
             msg :info, 'DHCP4v2::install'
 
             puts bash <<~SCRIPT
+                apk --no-cache add gcc musl-dev
```

Comment:
This is incorrect place for this I believe, this should be rather done in packer and then garbage collected :thinking: